### DISCUSSION TITLE
How did u get pro

### DISCUSSION THREAD

**Discussion by ghost** [KendallDoesCoding/KendallDoesCoding#12] [Q&A] [answered]
Hello kendal did u buy pro?

**Comment by KendallDoesCoding** [comment, marked as answer]
> Hello kendal did u buy pro?

No, I got it from the GitHub student plan @ https://education.github.com/

**Reply by ghost** [reply]
Well my school i live in india my school isn't listed there but it ask me to u know just give out about the school and the postal code and stuff and should i do that like will i be in trouble with my school bc i dont think they support it?

**Reply by KendallDoesCoding** [reply]
> Well my school i live in india my school isn't listed there but it ask me to u know just give out about the school and the postal code and stuff and should i do that like will i be in trouble with my school bc i dont think they support it?

You need to show proof from the school, I showed proof from the school without the school even knowing that I did it, so no you won't be in trouble.

My school is in india too

**Reply by ghost** [reply]
ok but what type of "proof " do i need ?????????

**Reply by KendallDoesCoding** [reply]
uh i forgot but you can check it out if you go further into applying

**Reply by KendallDoesCoding** [reply]
check this out: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack


u need to be 13+ to apply btw

**Reply by ghost** [reply]
> check this out: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack
> 
> u need to be 13+ to apply btw

im 14. so if i have a email which i use for school so i don't need any documents ? or do i need docs what docs ?

**Reply by KendallDoesCoding** [reply]
if you have a email given by the school for example, shour@school.edu then you need to upload less docs ig

**Reply by ghost** [reply]
k

**Comment by ghost** [comment]
sooo kendall so u just put ur school provided email and it gave u pro ? bc i did it but it ask me for acedemic proof or something idk what photo should i send @KendallDoesCoding

**Reply by KendallDoesCoding** [reply]
I put my personal email and attached school enrolment letter

**Reply by ghost** [reply]
ok
